opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -536,9 +536,9 @@ message ExponentialHistogramDataPoint {
     // Note: This uses a varint encoding as a simple form of compression.
     sint32 offset = 1;
 
-    // Count is an array of counts, where count[i] carries the count
-    // of the bucket at index (offset+i).  count[i] is the count of
-    // values greater than base^(offset+i) and less or equal to than
+    // BucketCounts is an array of count values, where BucketCounts[i] carries
+    // the count of the bucket at index (offset+i). BucketCounts[i] is the count
```

Comment:
```suggestion
    // bucket_counts is an array of count values, where bucket_counts[i] carries
    // the count of the bucket at index (offset+i). bucket_counts[i] is the count
```

Reply:
@open-telemetry/technical-committee since I cannot write suggestion and merge by myself (I can, but would be strange) would any of you look at this, apply suggestion and merge this PR?

Reply:
Done. I never used that button, please check the end result is what we need.